factory/processing/processComponents.go

```diff
@@ -585,11 +585,12 @@ func (pcf *processComponentsFactory) Create() (*processComponents, error) {
 	}
 
 	esdtDataStorageArgs := vmcommonBuiltInFunctions.ArgsNewESDTDataStorage{
-		Accounts:              pcf.state.AccountsAdapterAPI(),
-		GlobalSettingsHandler: disabled.NewDisabledGlobalSettingHandler(),
-		Marshalizer:           pcf.coreData.InternalMarshalizer(),
-		ShardCoordinator:      pcf.bootstrapComponents.ShardCoordinator(),
-		EnableEpochsHandler:   pcf.coreData.EnableEpochsHandler(),
+		Accounts:                      pcf.state.AccountsAdapterAPI(),
+		GlobalSettingsHandler:         disabled.NewDisabledGlobalSettingHandler(),
+		Marshalizer:                   pcf.coreData.InternalMarshalizer(),
+		ShardCoordinator:              pcf.bootstrapComponents.ShardCoordinator(),
+		EnableEpochsHandler:           pcf.coreData.EnableEpochsHandler(),
+		CrossChainTokenCheckerHandler: disabled.NewCrossChainTokenChecker(),
```

Comment:
Why is this disabled?

Reply:
The real component is not used here. It is used only in vm.

Reply:
I still don't think this is correct.

Reply:
created a real `crossChainTokenChecker` component